.editorconfig

```diff
@@ -0,0 +1,10 @@
+
+root = true
+
+[{src,scripts}/**.{ts,json,js}]
```

Comment:
I'm confused by this. Wouldn't it be `{src,test}`?  The `test` folder appears to have participated in the formatting.